### ISSUE TITLE
A key is permanently broken

### ISSUE THREAD

**Issue by R0quef0rt** [amark/gun#1063]
I am reporting a bug in the 0.2020.520 version of GUNDB, where something caused a specific key ('messages') to break indefinitely. I was using the following code at the time:
```
messages = gun.get('messages')
channel = messages.get('channels').get(targetChannel)
channel.get({ '.': { '>': timeStamp.format('YYYY/MM/DD hh:mm:ss') } }).map().map().on(async function(node, message){}
```
All of a sudden, the 'messages' key stopped working. The event listener just doesn't fire at all. If I change the key, and use the exact same code, things are fine. 

[Mark reports that he has seen this before, and that he thinks it's related to the lexical query](https://discord.com/channels/612645357850984470/612645357850984473/826063984036806656). However, removing the lex doesn't fix the problem. The 'messages' key appears to be broken for even the simplest of use-cases.

I fixed this by using a different key. But obviously, it's bad if a key can be broken forever.

**Comment by amark** [comment]
@R0quef0rt are you able to replicate this again, getting it to break?

**Comment by R0quef0rt** [comment]
Yes I am. I wouldn't know how to replicate the issue in a new environment, but my old one is still broken. I've moved to a different key at this point.

Here's one of three superpeers currently affected by it: https://resistance-tower.herokuapp.com/gun
